### DISCUSSION TITLE
alpha status

### DISCUSSION THREAD

**Discussion by smolinari** [marcj/deepkit#386] [General]
Hi,

Any plans to leave alpha status any time soon? I'd like to suggest the `type` library be used instead of `class-transformer` in another library, but I'm afraid the maintainer will say no, because the `type`library is still alpha.

Scott

**Comment by marcus-sa** [comment]
Yes, we're planning to leave alpha status in the upcoming weeks/months.

**Reply by smolinari** [reply]
Better late than never. :grin: 

Scott
